src/i_video.c

```diff
@@ -1206,3 +1206,9 @@ void I_BindVideoVariables(void)
     M_BindIntVariable("usegamma",                  &usegamma);
     M_BindIntVariable("png_screenshots",           &png_screenshots);
 }
+
+void I_ToggleFullScreen()
```

Comment:
Should be:

    void I_ToggleFullScreen(void)

In C, empty () parens in a function definition actually means "a function that can take any arbitrary number of arguments" (weird I know).